### PR TITLE
added Elyse ID photo

### PR DESCRIPTION
**Pull request by CurtainUp**
I can haz panda?

### CODE REVIEW

**Comment by Kazathur92** [comment]
sure you can ✌️ approved 👍

File: ids.html [hunk 1]
```diff
@@ -12,6 +12,11 @@
     <article>Noah Woods</article>
     <img src="https://static-cdn.jtvnw.net/jtv_user_pictures/hsdogdog-profile_image-5550ade194780dfc-300x300.jpeg" alt="Professor Dog">
   </div>
+
+  <div class="Elyse">
+    <h1>Elyse Dawson</h1>
+    <img src="https://vignette.wikia.nocookie.net/disney/images/3/35/PeterPanda.png/revision/latest?cb=20110914164045&path-prefix=es">
```
Comment (noahwoods) [inline ids.html]: please add an alt

**Comment by BryanNilsen** [comment]
👍

File: ids.html (hunk 1 above)
Comment (CurtainUp) [reply ids.html]: Added in latest commit. Yes! Keep things accessible for all users!

**Comment by CurtainUp** [comment]
Be sure to not only give a thumbs up, but approve your review in the top right Reviewers side bar. Thanks!

**Review by Kazathur92 — APPROVED** [review]

**Pull request merged**